### DISCUSSION TITLE
Expected PGS Scores - to Include Other Covaraites - (e.g. demography)?

### DISCUSSION THREAD

**Discussion by bgulko** [PGScatalog/pgsc_calc#109] [General]
Publications in the PGS catalog cite a variety of covariates (e.g., Sex, Age, Smoking, Demography PC's), presumably as removed confounders for identified variant impact. AUC and R^2 quoted in incremental form, but in some cases the trait variation described by removed covariates is also important. Is it anticipated that loading for these will become part of the harmonized profile / model at any point in the future?

This would allow an independent researcher to regenerate the full AUC for a trait (and quantify genetic demography!) from the harmonized model and improve usability.

Thanks!
--Brad

**Comment by nebfield** [comment]
I think @smlmbrt is the best person to answer your question, but he's on holiday and won't be back until the end of next week 🌴

**Reply by bgulko** [reply]
Thanks for the quick note, I'll look forward to hearing back!

**Comment by smlmbrt** [comment]
Hi @bgulko, thanks for your note. The performance metrics in the Catalog are there to give context for the predictive ability of the PGS - they are directly extracted from the initial publication and we try to match the key figures that are presented in the abstract and figures. The covariates and metrics are heterogenous because of this; however, we are working towards providing directly comparable performance metrics for PGS with a limited set of covariates (age, sex, PCs, etc) that may be more helpful in terms of the effect size (OR per SD increase in PGS), things like r2 and AUC are often more dependant on the study cohort, prevalence, etc. 

Could you explain a bit more what you mean by "trait variation described by removed covariates is also important"?

**Reply by bgulko** [reply]
Thanks, @smlmbrt. My goal is to convert polygenic scores into absolute risk, and ultimately for medically actionable use, using population statistics. This involves adjusting for differences in model development cohorts, or performance sizes, and also (ideally) inclusion of informative covariate risk (sex, age, and especially demographically-informative  PCs). AUC/R^2 in a demography provides one method of calibrating training set risk to population risk, but there are other approaches. A homogenous representation can dramatically reduce the amount of work needed to do this. I understand that this transformation requires accounting for a variety of types of summary performance metrics, as well as performance-population statistics. I'd love to be able to get this from a harmonized Catalog representation, rather than having to review each source.

**Reply by smlmbrt** [reply]
If I understand correctly this aligns with our motivation for curating the PGS effect sizes, which provide information of relative risk changes that can then be combined with demographic information to estimate absolute risk. If not, perhaps a specific example/mock-up of your ideal format would clarify.

**Reply by bgulko** [reply]
My understanding of the score interpretation continues to evolve, and my apologies in advance if this is redundant. I understand there are several issues in moving PGRS scores towards absolute risk, some of these are model-specific characteristics that it might be useful to have from the PGRS Catalog interface. Please let me know if these belong on other threads.

1)	My original question pertains to loadings ascribed to removed covariates (e.g., Sex, age, PC’s). While the variance attributed to these covariates seem of little use in identifying causal variants in a PGRS framework, they can be of use when attempting to verify the reported AUCs, and indeed reconstitute a more complete trait risk model that includes removed covariates. I was hoping they might be solicited by the PGRS Catalog for reporting in a harmonized way.

2)	Along with the topic of identifying model-specific data of use in generating absolute risk scores, it seems important to know the model development set size (Case/Control). Relative risk (or log odds) from PGRS scores are generally relative to the sizes of the development sets. If the relative training set sizes do not match the prevalence of the trait in a population then an adjustment is needed to develop absolute risk, that requires knowing those training sizes. This seems well reported in the supplementary information to PGP000262 (from e.g. PGS001790), but digging through the supplementary information for each paper to get this information seems contrary to the spirit of the online catalog.

3)	I am embarrassed to ask this, but some tests (e.g. PGS001830 or PGS002037) report performance in terms of partial correlation rather than log odds. I presume that the scores generated from such models/scores are real valued indicators that are correlated with presence of the trait but not necessarily the log-odds score. Is this correct? Perhaps the harmonized representation provides a correction, or I have misunderstood the result. If the score is not a log-odds measure, can you direct me towards a resource for converting from scores generated under Partial Correlate models?


Thanks again for your rapid responses. My use case is somewhat unusual, but I expect folks will seek more of it as time goes on.
